### ISSUE TITLE
[Bug] hugo languages.en.dateformat warnings on server

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass-website#469]
.. indeed still warnings.

While at improving the web pages, perhaps this could be addressed as well:

```
WARN 2024/08/30 17:25:25 config: languages.en.dateformat: custom params on the language top level is deprecated and will be removed in a future release. Put the value below [languages.en.params]. See https://gohugo.io/content-management/multilingual/#changes-in-hugo-01120

Re-run Hugo with the flag --panicOnWarning to get a better error message.
WARN 2024/08/30 17:25:25 config: languages.en.logo: custom params on the language top level is deprecated and will be removed in a future release. Put the value below [languages.en.params]. See https://gohugo.io/content-management/multilingual/#changes-in-hugo-01120

Re-run Hugo with the flag --panicOnWarning to get a better error message.
WARN 2024/08/30 17:25:25 config: languages.en.license: custom params on the language top level is deprecated and will be removed in a future release. Put the value below [languages.en.params]. See https://gohugo.io/content-management/multilingual/#changes-in-hugo-01120

Re-run Hugo with the flag --panicOnWarning to get a better error message.
WARN 2024/08/30 17:25:25 config: languages.en.socialicon: custom params on the language top level is deprecated and will be removed in a future release. Put the value below [languages.en.params]. See https://gohugo.io/content-management/multilingual/#changes-in-hugo-01120

Re-run Hugo with the flag --panicOnWarning to get a better error message.
Start building sites … 
hugo v0.113.0+extended linux/amd64 BuildDate=unknown
```

**Comment by veroandreo** [comment]
All those got fixed with #466, at least I do not get them any more when building locally with `hugo v0.113.0+extended`

**Comment by neteler** [comment]
Thanks, solved.
